### PR TITLE
Check if maxage header value is valid before setting it

### PR DESCRIPTION
**Pull request by senn**
Small but useful fix for issue #3251 

I'm hoping this could get some fast reviews and approvals, and even a fast release.  We could use this fix asap.

### CODE REVIEW

File: bundle/src/main/java/com/adobe/acs/commons/http/headers/impl/PropertyBasedDispatcherMaxAgeHeaderFilter.java [hunk 1]
```diff
@@ -100,7 +100,9 @@ protected String getHeaderValue(HttpServletRequest request) {
             Resource resource = getResource(slingRequest);
             if (resource != null) {
                 String headerValue = resource.getValueMap().get(propertyName, String.class);
-                return HEADER_PREFIX + headerValue;
+                if (isValidMaxAgeValue(headerValue)) {
+                    return HEADER_PREFIX + headerValue;
+                }
```
Comment (kwin) [inline bundle/src/main/java/com/adobe/acs/commons/http/headers/impl/PropertyBasedDispatcherMaxAgeHeaderFilter.java:105]: Some additional logging on debug level would be helpful here

**Review by kwin — APPROVED** [review]
LGTM, thanks.

**Comment by henrykuijpers** [comment]
Indeed, looks very good! Can this be merged quickly, would be nice to have it in the next release @kwin  @davidjgonzalez

**Comment by davidjgonzalez** [comment]
@senn  can you add a Changelog entry?

**Comment by senn** [comment]
Thx for the fast actions, everybody!

@davidjgonzalez I see you have already added the changelog yourself. ✅

**Pull request merged**